nemo/collections/nlp/models/language_modeling/megatron_gpt_sft_model.py

```diff
@@ -541,46 +593,88 @@ def inference_epoch_end(self, outputs, mode, data_cfg):
             averaged_metric = 0.0 if monitor_mode == 'max' else 1e5
 
         if mode == 'validation':
-            self.log("validation_loss", averaged_loss)
+            self.log("validation_loss", averaged_loss, batch_size=1, sync_dist=True)
```

Comment:
may not be necessary, I added `batch_size=1, sync_dist=True` just because other places have them. it's ok to keep the original code.